### PR TITLE
Adds fields, recurring, and scheduledTime

### PR DESCRIPTION
**Pull request by adamaltman**

### CODE REVIEW

File: spec/definitions/DataExport.yaml [hunk 1]
```diff
@@ -47,13 +47,45 @@ properties:
...
+    required:
+      - start
+      - end
   emailNotification:
     description: List of emails to be notified when export is completed
     type: array
     items:
       type: string
       format: email
+  fields:
```
Comment (XaosSintez) [inline spec/definitions/DataExport.yaml]: I wonder if there will be all fields by default (if this field's value is skipped). In this case you should note that in the descriptions so it will be not confusing

Comment (adamaltman) [reply spec/definitions/DataExport.yaml]: It will not be all by default.  It will be whatever our default fields are.

Comment (adamaltman) [reply spec/definitions/DataExport.yaml]: Let me know if the sentence I added clarifies that at all.

File: spec/definitions/DataExport.yaml [hunk 2]
```diff
@@ -47,13 +47,45 @@ properties:
...
+    description: List of fields to include in the export.  See supporting documentation for more info.  If omitted, default fields will be included.
+    type: array
+    items:
+      type: string
+  recurring:
```
Comment (pmeth) [inline spec/definitions/DataExport.yaml]: Can this be put into a separate PR since scheduling is not working right now?

Comment (adamaltman) [reply spec/definitions/DataExport.yaml]: We want to provide final documentation.  So it's okay if it doesn't work at the moment.
